packages/assistant/src/RPA/Assistant/date_picker.py

```diff
@@ -0,0 +1,87 @@
+from flet import Dropdown, UserControl, Container, Row, Icon, PopupMenuButton, PopupMenuItem, Text
+from flet.dropdown import Option
+from flet.icons import CALENDAR_MONTH, ARROW_BACK_ROUNDED, ARROW_FORWARD_ROUNDED
+from flet import MainAxisAlignment
+from datetime import datetime, date
+from calendar import monthrange
```

Comment:
Nit: `isort` tool can be used here to sort deps into standard, 3rd-party and 1st-party groups.

Reply:
Good point, I'll have to run the related invoke commands for docs and linting

Reply:
done